recipe/recipe.yaml

```diff
@@ -0,0 +1,146 @@
+# TODO: Update package pinnings in conda_build_config for every release
+schema_version: 1
+context:
+  version: 1.49.0
+package:
+  name: nvidia-dali-python
+  version: ${{ version }}
+source:
+  - url: https://github.com/NVIDIA/DALI/archive/refs/tags/v${{ version }}.tar.gz
+    sha256: da4f2d2a59ae3d9e08c4004b16fdd36bd8585474381c215cbd649c76daeea717
+    patches:
+      - patches/0001-BLD-Use-shared-library-deps.patch
+      - patches/0002-BLD-Find-nvimgcodec-package-instead-of-downloading-s.patch
+      - patches/0003-BLD-Use-CUDA-target-include-directory-to-support-cro.patch
+      - patches/0004-BUG-Add-missing-header-for-std-runtime_error.patch
+      - patches/0005-BLD-Link-to-cudart-statically.patch
+  - url: https://github.com/cocodataset/cocoapi/archive/8c9bcc3cf640524c4c20a9c40e89cb6a2f2fa0e9.tar.gz
+    sha256: 4dd3450bab2287d1c1b271cf4a1420db05294194f0ad9af4e3af592d6f2b4410
+    target_directory: third_party/cocoapi
+  - url: https://github.com/JanuszL/ffts/archive/6ef8d818f46f679c95110ab199ae915fb04bfef5.tar.gz
+    sha256: 1d157157828ce1c85d0a43a6de312ce168b3c5dc1223c6ad3f02e074c3806c15
+    target_directory: third_party/ffts
+build:
+  number: 0
+  skip:
+    - cuda_compiler_version == "None"
+    - win
+    # - match(python, "!=3.12.*")
+    # nvjpegBufferPinnedCreate is required but not present
+    - linux and aarch64 and match(cuda_compiler_version, "<12")
+requirements:
+  ignore_run_exports:
+    by_name:
+      - cuda-version
+    from_package:
+      - if: match(cuda_compiler_version, "11.*")
+        then:
+          - cudatoolkit
+      - if: match(cuda_compiler_version, "12.*")
+        then:
+          - libcufft-dev
+          - libcufile-dev
+          - libcurand-dev
+          - libnpp-dev
+          - libnvjpeg-dev
+      - libnvimgcodec-dev  ${{ libnvimgcodec }}
+  build:
+    - ${{ compiler('cuda') }}
+    - ${{ compiler('cxx') }}
+    - ${{ stdlib('c') }}
+    - cmake 3.*
+    - ninja
+    - if: build_platform != target_platform
+      then:
+        - cross-python_${{ target_platform }}
+        - python
+        - python-clang
+  host:
+    - if: match(cuda_compiler_version, "11.*")
+      then:
+        - cudatoolkit
+    - if: match(cuda_compiler_version, "12.*")
+      then:
+        - cuda-cudart-static
```

Comment:
How is the license requirements satisfied when this library is statically linked?

Reply:
It's not.

- [ ] Add NVIDIA license or dynamically link cudart